hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneClientAdapterImpl.java

```diff
@@ -534,7 +536,10 @@ private FileStatusAdapter toFileStatusAdapter(OzoneFileStatus status,
         owner,
         owner,
         null,
-        getBlockLocations(status)
+        getBlockLocations(status),
+        !Objects.isNull(keyInfo.getFileEncryptionInfo()),
+        keyInfo.getReplicationConfig().getReplicationType() ==
+        HddsProtos.ReplicationType.EC
```

Comment:
how about creating a helper method isErasureCoded() in OzoneClientAdpter and move this code there so that this code can be shared by both BasicOzoneClientAdapterImpl and BasicOzoneRootedOzoneClientAdapterImpl?